### ISSUE TITLE
[Problem] [PartDesign] [SubObject Binder] Binder edges are invisible in sketch, if overlapping with Axis

### ISSUE THREAD

**Issue by V5Studio** [FreeCAD/FreeCAD#10334]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Full version info

```shell
[code]
OS: Windows 11 build 22621
Word size of FreeCAD: 64-bit
Version: 0.21.0.33675 (Git)
Build type: Release
Branch: (HEAD detached at 0.21.0)
Hash: 41b058e2087de60dc8fef3d3e68c7d0129e13abf
Python 3.10.12, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.5, OCC 7.6.3
Locale: English/United States (en_US)
Installed mods: 
  * 3DfindIT 1.2.0
  * freecad.gears 1.0.0
  * parts_library
[/code]
```

### Subproject(s) affected?

PartDesign

### Problem description

Created a simple 3 edge binder

![image](https://github.com/FreeCAD/FreeCAD/assets/112349172/26bba25c-f7d3-47b8-82cd-7ce193d7f4ea)

Moving to Sketch, the Axis seems to overlap the edge making it impossible to see

![image](https://github.com/FreeCAD/FreeCAD/assets/112349172/1c67ccb8-58d3-4fb0-98f9-19fdc46b2135)

### Anything else?

[Binder Issue.zip](https://github.com/FreeCAD/FreeCAD/files/12422097/Binder.Issue.zip)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Syres916** [comment]
Yes, this is the rendering order, while a fix is done, there is a workaround by @0penBrain which removes the axes just in the current Sketcher Edit Mode (they are redrawn after leaving and re-entering Edit Mode), see https://forum.freecad.org/viewtopic.php?style=3&p=502114#p502114 to generate a macro which you can add to either the Sketcher or Global toolbar for example.

**Comment by V5Studio** [comment]
Thanks @Syres916 for a quick solution. I am curious though, shouldn't the axes always be the first to be drawn in the render order?

**Comment by Syres916** [comment]
>  I am curious though, shouldn't the axes always be the first to be drawn in the render order?

They are the first to be drawn in a new sketch but if you draw a Normal Geometric line on top of one and then draw a construction line on top of that then there needs to be am order top-down. At present objects outside the sketch have the lowest priority, then the axes and then the user can select the order or Normal, Construction and External Geometry.

**Comment by V5Studio** [comment]
Thanks for enlightening me.. I guess the axes should be considered special case and should always be drawn first!

**Comment by FEA-eng** [comment]
Still relevant

```
OS: Windows 11 build 26100
Architecture: x86_64
Version: 1.1.0dev.14555 (Git shallow)
Build date: 2025/09/09 06:43:51
Build type: Release
Branch: main
Hash: c1dd116e1e3304511d9e5e634e711e549b2b84e7
Python 3.11.13, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL)
Navigation Style/Orbit Style/Rotation Mode: Blender/Rounded Arcball/Window center
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
Logical DPI/Physical DPI/Pixel Ratio: 96/81.5973/1
```
